src/components/blog/QuoteBlock/QuoteBlock.tsx

```diff
@@ -7,7 +8,7 @@ type QuoteBlockProps = {
 
 const QuoteBlock = ({ quote, attributee }: QuoteBlockProps) => {
   const quoteContent = attributee ? `${quote} - ${attributee}` : quote;
-  return <p>{quoteContent}</p>;
+  return <p className={s.quote}>{quoteContent}</p>;
```

Comment:
I ended up using the `<q>` element instead, because `<blockquote>` isn't inline, so there was no good way for me to get the attributee on the same line.

Reply:
Sounds good. You probably could have just set the display style to `inline`, but I think this is fine, too.